addons/sys_radio/fnc_monitorRadios.sqf

```diff
@@ -195,7 +195,12 @@ DFUNC(hasGearDesync) = {
         {
             _message = _message + format ["%1\n", name _x];
         } forEach ACRE_SERVER_DESYNCED_PLAYERS;
-        hintSilent _message;
+        if(GVAR(disableDesyncHint)) then {
+            diag_log _message;
```

Comment:
As said on Slack, this should always be printed to RPT, even if it's hinted. And also using a macro. So this entire thing would become:
```
WARNING(_message);
if (!GVAR(disableDesyncHint)) then {
    hintSilent _message;
};
```